src/views/Events/Events.module.scss

```diff
@@ -0,0 +1,25 @@
+.event_page {
+  &_searchPageTitle {
```

Comment:
Same thing here, we should get rid of the & nested classes where they aren't needed.